### ISSUE TITLE
传入的contentView使用xib layout 布局时内容撑不开（例：多行UILabel不会自动撑开，需要设置准确的contentView高度才可以）这种情况怎么设置

### ISSUE THREAD

**Issue by RS-Li** [JonyFang/FFPopup#42]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](../CONTRIBUTING.md)
* [ ] I have searched for a similar issue in the [project](https://github.com/JonyFang/FFPopup/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios
 Platform Version        | e.g. 11.0 / 12.1.2
 FFPopup Version         | e.g. 1.0.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 10.1
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.
